app/src/main/java/com/jerboa/ui/components/comment/mentionnode/CommentMentionNode.kt

```diff
@@ -54,16 +55,21 @@ fun CommentMentionNodeHeader(
     personMentionView: PersonMentionView,
     onPersonClick: (personId: PersonId) -> Unit,
     score: Long,
+    upvotes: Long,
+    downvotes: Long,
     myVote: Int,
     onClick: () -> Unit,
     onLongClick: () -> Unit,
     showAvatar: Boolean,
-    showScores: Boolean,
+    voteDisplayMode: VoteDisplayMode,
 ) {
     CommentOrPostNodeHeader(
         creator = personMentionView.creator,
         score = score,
         myVote = myVote,
+        upvotes = upvotes,
+        downvotes = downvotes,
+        voteDisplayMode = voteDisplayMode,
```

Comment:
Passing the whole InstantScores basically here? Why not pass the InstantScores itself then?

Reply:
Mmmk, I'll refactor all these to do instant scores.